### PR TITLE
ocrvs-4366: Delete integration

### PR DESCRIPTION
**Pull request by sifulovi**
### For Deletion
[delete.webm](https://user-images.githubusercontent.com/11840131/204783391-2bea535a-3efc-4ef3-bb89-8d077f3e29ea.webm)

### For copy to clip

https://user-images.githubusercontent.com/11840131/205293461-1e0b4803-9045-4cd1-a0ef-afd4a33cb5bd.mp4

### Story for CopyButton 

https://user-images.githubusercontent.com/11840131/205607045-61834fc4-6af1-47a4-be0c-445d71760ca8.mp4

### CODE REVIEW

File: packages/user-mgnt/src/features/system/handler.ts [hunk 1]
```diff
@@ -505,3 +505,26 @@ export async function refreshSystemSecretHandler(
...
+    const { clientId } = request.payload as SystemClientIdPayload
+
+    const system = await System.findOneAndDelete({
+      client_id: clientId
+    })
+
+    if (system) {
+      logger.info(`System has been deleted by clientId ${clientId}`)
+      return h.response(pickSystem(system!)).code(200)
+    }
```
Comment (Zangetsu101) [inline packages/user-mgnt/src/features/system/handler.ts:523]:
```suggestion
    if (system) {
      logger.info(`System has been deleted by clientId ${clientId}`)
      return h.response(pickSystem(system)).code(200)
    }
```

File: packages/client/src/views/SysAdmin/Config/Systems/Copy.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+interface CopyProps {
+  data: string
+}
+
+export function Copy({ data }: CopyProps) {
+  const [clipToCopy, setClipToCopy] = useCopyToClipboard()
+  const intl = useIntl()
+  return (
+    <TertiaryButton
```
Comment (naftis) [inline packages/client/src/views/SysAdmin/Config/Systems/Copy.tsx:29]: We should rather use `<Button type="tertiary">`

File: packages/client/src/views/SysAdmin/Config/Systems/Systems.tsx [hunk 3]
```diff
@@ -75,6 +77,13 @@ const StyledSpinner = styled(Spinner)`
 const Field = styled.div`
   margin-top: 16px;
 `
+
+const Details = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
```
Comment (naftis) [inline packages/client/src/views/SysAdmin/Config/Systems/Systems.tsx:85]: i think we could achieve this with `<Stack>` component

**Review by naftis — CHANGES_REQUESTED** [review]
small changes but looks good otherwise

**Review by naftis — APPROVED** [review]

**Pull request merged**